src/main/webapp/app/exam/participate/exam-bar/exam-bar.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ArtemisSharedCommonModule } from 'app/shared/shared-common.module';
+import { ArtemisExamTimerModule } from 'app/exam/participate/timer/exam-timer.module';
+import { ArtemisExamLiveEventsModule } from 'app/exam/participate/events/exam-live-events.module';
+import { ExamParticipationService } from 'app/exam/participate/exam-participation.service';
+import { Exercise, ExerciseType } from 'app/entities/exercise.model';
+import { IconProp } from '@fortawesome/fontawesome-svg-core';
+import { faDoorClosed } from '@fortawesome/free-solid-svg-icons';
+
+import dayjs from 'dayjs/esm';
+
+@Component({
+    selector: 'jhi-exam-bar',
+    standalone: true,
+    imports: [CommonModule, ArtemisSharedCommonModule, ArtemisExamTimerModule, ArtemisExamLiveEventsModule],
+    templateUrl: './exam-bar.component.html',
+    styleUrl: './exam-bar.component.scss',
+})
+export class ExamBarComponent {
+    @Output() onExamHandInEarly = new EventEmitter<void>();
+    @Output() examAboutToEnd = new EventEmitter<void>();
+
+    @Input() examTitle: string;
+    @Input() examTimeLineView = false;
+    @Input() endDate: dayjs.Dayjs;
+    @Input() exerciseIndex = 0;
+    @Input() exercises: Exercise[] = [];
+
+    icon: IconProp = faDoorClosed;
```

Comment:
The icon can be readonly, usually we just keep the name of the icon - but I do not have a strong opinion here (so if you prefer `icon` I do not mind)
```suggestion
    readonly faDoorClosed: IconProp = faDoorClosed;
```